tasks/gitlab_permissions.yml

```diff
@@ -5,14 +5,19 @@
     owner: git
     recurse: yes
 
+- name: Read default config
+  include_vars:
+    file: "gitlab.yml.defaults"
+    name: gitlab_config_default
+
 - name: Place GitLab config
-  template: 
-    src: templates/gitlab.yml.j2
+  copy: 
+    content: "{{ gitlab_config_default | combine(gitlab_config, recursive=True) | to_nice_yaml }}"
     dest: "{{ gitlab_install_path | mandatory }}/config/gitlab.yml"
```

Comment:
What about owner, rights and groups? For all files basically.

Reply:
i set the permission on files that have to be writable by gitlab otherwise they belong to root